src/components/Button/Button.js

```diff
@@ -16,7 +17,9 @@ const Button = ({
       className={cx(
         "button",
         { "button--fluid": fluid },
-        { "button--style-reset": styleReset }
+        { "button--style-reset": styleReset },
+        { "button--info": theme === "info" },
```

Comment:
With multiple themes to the buttons, I would add a `primary` theme for the default buttons, and default the `theme` prop to `primary` and then apply the classes with

```
cx("button", `button--${theme}`, { your conditional styles })
```